File: 1919.cpp
#include <iostream>
#include <string>
#include <cmath>
#define SIZE 26
using namespace std;

int main(){
    cin.tie(0);
    ios_base::sync_with_stdio(0);
    string s1, s2;
    cin >> s1 >> s2;
    int arr[SIZE] = {0};
    for (auto c:s1) arr[c - 'a']++;
    for (auto s:s2) arr[s - 'a']--;
    int sum = 0;
    for (int i = 0; i < SIZE; i++){
        if (arr[i] != 0){
            sum += abs(arr[i]);
        }
    }
    cout << sum;
    return 0;
}

File: 1919.py
str1 = str(input())
str2 = str(input())

arr = [0 for _ in range(26)]

for i in range(len(str1)):
    arr[ord(str1[i]) - ord('a')] += 1

for i in range(len(str2)):
    arr[ord(str2[i]) - ord('a')] -= 1

sum = 0
for i in arr:
    if (i != 0):
        sum += abs(i)

print(abs(sum))
